dev/storage.h

```diff
@@ -774,301 +792,93 @@ namespace sqlite_orm {
             template<class T, class... Args>
             prepared_statement_t<select_t<T, Args...>> prepare(select_t<T, Args...> sel) {
                 sel.highest_level = true;
-                auto con = this->get_connection();
-                sqlite3_stmt *stmt;
-                auto db = con.get();
-                using context_t = serializator_context<impl_type>;
-                context_t context{this->impl};
-                context.skip_table_name = false;
-                context.replace_bindable_with_question = true;
-                auto query = serialize(sel, context);
-                if(sqlite3_prepare_v2(db, query.c_str(), -1, &stmt, nullptr) == SQLITE_OK) {
-                    return {std::move(sel), stmt, con};
-                } else {
-                    throw std::system_error(std::error_code(sqlite3_errcode(db), get_sqlite_error_category()),
-                                            sqlite3_errmsg(db));
-                }
+                return prepare_impl<select_t<T, Args...>>(std::move(sel));
```

Comment:
tip: it is not necessary to specify template argument explicitly in this case. One can write `return prepare_impl(std::move(sel));` and that will work